source/frontend/src/features/leases/detail/LeasePages/LeasePageForm.tsx

```diff
@@ -13,11 +19,41 @@ export interface ILeasePageFormProps {
  */
 export const LeaseViewPageForm: React.FunctionComponent<
   React.PropsWithChildren<ILeasePageFormProps>
-> = ({ children, isEditing, onEdit }) => {
+> = ({ children, leasePageName, isEditing, onEdit }) => {
+  const { lease } = useContext(LeaseStateContext);
+
+  const DisplayLeaseTerminationMessage = () => {
```

Comment:
can this be simplified to a single statement?

`
return (lease && leasePageName === LeasePageNames.DETAILS) && (lease.fileStatusTypeCode.id === ApiGen_CodeTypes_LeaseStatusTypes.DISCARD || lease.fileStatusTypeCode.id === ApiGen_CodeTypes_LeaseStatusTypes.TERMINATED)
`

Reply:
Updated